src/app/auth/oauth.py

```
@@ -2,16 +2,15 @@
+import logging
+from typing import Tuple, Optional
+from flask import url_for, redirect, current_app
 from ..config import settings
 
-try:  # pragma: no cover - dependency may be unavailable during tests
-    import mwoauth  # type: ignore[import-not-found]
-except ModuleNotFoundError:  # pragma: no cover - handled when functions are used
-    mwoauth = None  # type: ignore[assignment]
+logger = logging.getLogger(__name__)
+
+IDENTITY_ERROR_MESSAGE = "We couldn’t verify your MediaWiki identity. Please try again."
```

Ruff (0.14.1)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)